File: WordSwell.Frontend/src/Faculty/Backend/BoardCont/BoardSearchTargetType.ts
/** 검색 대상 */
export const enum BoardSearchTargetType 
{
    /** 없음 */
    None = 0,
    /** 게시물 고유번호 */
    PostIndex = 1,
    /** 제목 */
    Title = 2,
    /** 내용 */
    Contents = 3,
    /** 제목 + 내용 */
    TitleAndContents = 4,
    /** 유저 이름 */
    UserName = 5,
}

File: WordSwell.ApiModels/BoardCont/BoardSearchTargetType.cs
namespace WordSwell.ApiModels.BoardCont;

/// <summary>
/// 검색 대상
/// </summary>
public enum BoardSearchTargetType
{
    /// <summary>
    /// 없음
    /// </summary>
    None = 0,

    /// <summary>
    /// 게시물 고유번호
    /// </summary>
    PostIndex,

    /// <summary>
    /// 제목
    /// </summary>
    Title,

    /// <summary>
    /// 내용
    /// </summary>
    Contents,

    /// <summary>
    /// 제목 + 내용
    /// </summary>
    TitleAndContents,

    /// <summary>
    /// 유저 이름
    /// </summary>
    UserName,
}
